virt/virt-2-4-release-notes.adoc

```diff
@@ -12,10 +12,17 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == New and changed features
 //Placeholder for new content.
 
-//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing
+//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing - Jira story was obsoleted.
 
 //CNV-4687 - Ensure platform passes Windows Server Virtualization Program with RHCOS workers.
 
+* {VirtProductName} is certified in Microsoft's Windows Server
+Virtualization Validation Program (SVVP) to run Windows Server workloads.
+** SVVP Certification applies to Intel and AMD CPUs.
+** SVVP Certified Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
```

Comment:
I'm unsure if it should be `Cerified` or `certified`

Reply:
Content was revamped/updated in the later commit.